### DISCUSSION TITLE
I want

### DISCUSSION THREAD

**Discussion by miso1459** [oqtane/oqtane.framework#1169] [Q&A] [answered]
I want to debug the "oqtane.blogs" with Visual Studio 2019.
Please tell me how.

Thank you.

**Comment by leigh-pointer** [comment, marked as answer]
Hi @miso1459 have a read of this blog entry [https://blazorhelpwebsite.com/ViewBlogPost/15](https://blazorhelpwebsite.com/ViewBlogPost/15) there you will find a section on Debugging.

Hope this helps

**Reply by miso1459** [reply]
Thank you~~~
